infra/base-images/base-builder/Dockerfile

```diff
@@ -41,6 +41,8 @@ ENV FUZZER_LDFLAGS ""
 
 WORKDIR $SRC
 
+RUN git clone https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer libfuzzer
```

Comment:
Should we clone libFuzzer here (similar to other engines) instead of dong that in `base-clang` image? I haven't found any reason to do it there, looks like a leftover from the times when we supported libFuzzer only :)